deploy/cloudformation/elastic-agent-ec2.yml

```diff
@@ -68,6 +68,10 @@ Resources:
             - ebs:ListChangedBlocks
             - ebs:GetSnapshotBlock
             Resource: "*"
+          - Effect: Allow
```

Comment:
Nice!

Reply:
So before this change `cloud.account.name` and the other fields weren't populated?